pkg/migrations/op_drop_constraint.go

```diff
@@ -122,6 +122,10 @@ func (o *OpDropConstraint) Rollback(ctx context.Context, conn *sql.DB) error {
 }
 
 func (o *OpDropConstraint) Validate(ctx context.Context, s *schema.Schema) error {
+	if err := validateName(o.Column); err != nil {
+		return err
+	}
+
```

Comment:
I don't think it makes sense to check the length of the column name on the operation here as it must already exist in the database. The check can be removed.